doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,135 @@
+
+* `+` indicates that the feature is present in the test scenario designer.
+* `-` indicates that the feature is not present in the test scenario designer.
+
+.Main features of legacy and new test scenario designer
+[cols="30%,30%,30%,30%", options="header"]
+|===
+|Feature & highlights
+|New test scenario designer
+|Legacy test scenario designer
+|Documentation
+
+a|*Creating and running a test scenario*
+
+* You can create test scenarios in {CENTRAL} to test the functionality of business rule data before deployment.
+* A basic test scenario must have at least a related data objects, *GIVEN* facts or *EXPECT* results.
+* You can run the tests to validate your business rules and data.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about creating rule and DMN-based test scenarios, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about running the test scenarios, see xref:test-designer-run-test-proc[].
+* For more information about creating and running test scenarios(legacy), see xref:test-scenarios-legacy-create-proc[].
+
+a|*Adding GIVEN facts in test scenarios*
+
+You can insert and verify the *GIVEN* facts for the test.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about adding GIVEN facts in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding GIVEN facts in test scenarios(legacy), see xref:test-scenarios-legacy-GIVEN-proc[].
+
+a|*Adding EXPECT results in test scenarios*
+
+* It represents the objects and their fields whose exact values are checked based on the given information.
+* The *EXPECT* section defines the expected results based on the *GIVEN* input facts.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about adding EXPECT results in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding EXPECT results in test scenarios(legacy), see xref:test-scenarios-legacy-EXPECT-proc[].
+
+a|*KIE session & KIE base for testing*
+
+* KIE session can be set on test scenario level settings.
+* KIE base can be set on project level settings.
+
+|`+`
+|`+`
+|-
+
+|*Simulated date and time*
+|`-`
+|`+`
+|-
+
+a|*Rule flow group*
+
+You can specify a rule flow group to be activated so that all rules within that group can be tested.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about setting rule flow group in new test scenarios, see xref:test-designer-global-settings-panel-rule-based-proc[].
+* For more information about setting rule flow group in test scenarios(legacy), xref:test-scenarios-legacy-GIVEN-proc[].
+
+a|*Globals*
+
+* Global variables are named objects that are visible to the decision engine but are different from the objects for facts.
+* Global variables are deprecated for new test scenarios.
```

Comment:
I would say "Setting global variables are deprecated for new test scenarios."
Or something along those lines, since global variables still exist in the DRL, but you just can't set them